corehq/apps/app_manager/static/app_manager/js/releases.js

```diff
@@ -143,7 +143,8 @@ function ReleasesMain(o) {
     self.download_application_zip = function(appId, multimedia_only) {
         var url_slug = multimedia_only ? 'download_multimedia' : 'download_zip';
         var url = self.url(url_slug, appId);
-        self.async_downloader.generateDownload(url);
+        message = "Your application download is ready";
```

Comment:
the one bummer about this is that this won't be translated. not gonna block on it since there are other messages in this file that also don't translate